src/Pact/Repl/Lib.hs

```diff
@@ -588,16 +588,20 @@ setGasPrice i as = argsError i as
 
 envExecConfig :: RNativeFun LibState
 envExecConfig i as = case as of
-  [TLitBool allowMod,TLitBool allowHist] -> do
-    let ec = ExecutionConfig allowMod allowHist
+  [TList reps _ _] -> do
+    fs <- forM reps $ \s -> case s of
+      TLitString r -> case M.lookup r flagReps of
+        Nothing -> evalError' i $ "Invalid flag, allowed: " <> viaShow (M.keys flagReps)
```

Comment:
Can we add a Repl test to exercise this code path

Reply:
Nah, it's just repl stuff